src/MessagingInput/MessagingInput.tsx

```diff
@@ -31,6 +31,10 @@ interface Props {
   labelText?: string;
   disableSendButton?: boolean;
   showReturnKeyInstructions?: boolean;
+  // TODO: is this needed or is the existance of MessagingInput.store sufficient to decide this?
```

Comment:
I have no strong preference, can see an argument to be made for both cases (more explicit vs less props)

Reply:
I'm gonna go with more explicit for now, will remove comment